src/mjolnir/graphbuilder.cc

```diff
@@ -376,14 +375,18 @@ void BuildTileSet(const std::string& ways_file, const std::string& way_nodes_fil
 
   auto database = pt.get_optional<std::string>("admin");
   // Initialize the admin DB (if it exists)
-  sqlite3 *admin_db_handle = GetDBHandle(*database);
-  if (!admin_db_handle)
+  sqlite3 *admin_db_handle = database ? GetDBHandle(*database) : nullptr;
```

Comment:
every where where we load sqlite i made the config optional because actually having it is optional as well, simplifies peoples lives who dont put this in the config